cmd/daemon/main.go

```diff
@@ -23,14 +30,19 @@ func main() {
 		log.Fatal(err)
 	}
 
-	ctrd, err := freezer.Connect()
+	fre, err := freezer.GetFreezer(runtimeType)
```

Comment:
total nit, but can we have this be `freezer` so it matches `thawer`?

Reply:
yeah,of course, first let me rename the package `freezer` to `freeze` to avoid conficts